browser/ui/BUILD.gn

```diff
@@ -537,6 +537,8 @@ source_set("ui") {
       "views/rounded_separator.cc",
       "views/rounded_separator.h",
       "views/side_panel/ai_chat/ai_chat_side_panel_utils_views.cc",
+      "views/side_panel/ai_chat_side_panel_web_view.cc",
```

Comment:
why are these not in ai_chat? Also these should go in their own BUILD.gn and use brave_ui_allow_circular_includes_from